Changelog.md

```diff
@@ -3,6 +3,7 @@
 ## 0.0.34
 
 - Provide hover for Kamelet properties based on embedded snapshot Kamelet catalog version
+- Update Camel Kafka Connector Catalog from 0.9.0 to 0.10.1

```

Comment:
Might be clearer by adding "version" -- i.e. "Update Camel Kafka Connector Catalog version from 0.9.0 to 0.10.1"